src/modules/commentPopup.js

```diff
@@ -0,0 +1,69 @@
+import { fetchSingleProduct, getComments } from './commentAPI.js';
+import {
+  displayContent,
+  displayComments,
+  drawStars,
+  changeThumbnails,
+} from './displayPopup.js';
+import submitComment from './commentSubmit.js';
+import Counter from './count.js';
+
+const Count = new Counter();
+
```

Comment:
- [ ] It would be better if you can close the popup without having to refresh the app or comment on the article. Please add a way to close the app. 😄 👍